src/components/static/Countdown.jsx

```diff
@@ -65,9 +67,11 @@ function calculateBlocksForDuration(duration) {
  * @return {JSX.Element}
  */
 const Countdown = ({ targetTime }) => {
-  const [blocks, setBlocks] = useState(
-    calculateBlocksForDuration(Math.max(targetTime - Date.now(), 0))
-  );
+  const [blocks, setBlocks] = useState([]);
+  // Cypress E2E CI keeps dying... https://nextjs.org/docs/messages/react-hydration-error
+  useEffect(() => {
+    setBlocks(calculateBlocksForDuration(Math.max(targetTime - Date.now(), 0)));
```

Comment:
remove the comment, why is this in use effect?

Reply:
When you initialize it in useState parameter, it'll try to prerender, but this results in a server-client mismatch (react error 425) in the e2e testing stage. I'll see what I can do to make it not load a placeholder initially while still passing e2e tests.